### PR TITLE
Fix button collector - 'button is not defined'

### PR DESCRIPTION
**Pull request by RagnarLothbrok-Odin**
When clicking a button with a collector, you get an error of `button is not defined`
I defined the button correctly.

### CODE REVIEW

**Comment by AngeloCore** [comment]
Change the branch to `dev`

**Comment by RagnarLothbrok-Odin** [comment]
Done!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "discord-buttons",
-  "version": "3.2.1-dev",
+  "version": "3.2.1",
```
Comment (AngeloCore) [inline package.json]: plese change the version to `3.2.5-dev`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "discord-buttons",
-  "version": "3.2.1",
+  "version": "3.2.1-dev",
```
Comment (AngeloCore) [inline package.json]: change the version to `3.2.5-dev`

**Review by AngeloCore — CHANGES_REQUESTED** [review]

**Comment by RagnarLothbrok-Odin** [comment]
Done.

**Pull request merged**